service/collector/app_resource.go

```diff
@@ -1,13 +1,31 @@
 package collector
 
 import (
+	"context"
+	"fmt"
+
 	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	"github.com/prometheus/client_golang/prometheus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 )
 
+var (
+	appDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "app"),
```

Comment:
So this is `app_operator_app`? I'd expect just `app_info`

Reply:
Good catch, I'll change the name in here suffix from `app` into `info`

Reply:
The `_info` suffix is useful for these kinds of metrics, so i'd like it to be `app_info`